<launch>
  <!-- example calibration launch file for kuka iiwa project -->

  <node name="handeye_calibration_server" type="calibration_server.py" pkg="simple_handeye_calibration" output="screen">
    <param name="camera_marker" value="ar_marker_0"/>
    <param name="ee_marker" value="endpoint_marker"/>
    <param name="base_link" value="world"/>
    <param name="camera_link" value="camera_link"/>
  </node>

</launch>
